test/navigation/ReturnsJourneyNavigatorSpec.scala

```diff
@@ -18,111 +18,141 @@ package navigation
 
 import base.SpecBase
 import controllers.returns.{routes => returnsRoutes}
+import controllers.returns.credits.{routes => creditsRoutes}
 import models.Mode.{CheckMode, NormalMode}
+import org.scalatestplus.play.PlaySpec
 
-class ReturnsJourneyNavigatorSpec extends SpecBase {
+class ReturnsJourneyNavigatorSpec extends PlaySpec {
 
   private val returnsJourneyNavigator = new ReturnsJourneyNavigator
   val navigator = new Navigator(returns = returnsJourneyNavigator)
 
   //todo add in credits navigation that is currently tested in controllers
```

Comment:
Made a start with these credits pages, looks like a lot of other routing defs are untested aswell